terraso_backend/apps/graphql/schema/users.py

```diff
@@ -4,11 +4,18 @@
 from graphene_django import DjangoObjectType
 
 from apps.core.models import User
+from apps.storage.services import ProfileImageService
 
 from .commons import BaseDeleteMutation
 
 
 class UserNode(DjangoObjectType):
+    profile_image = graphene.String(description="The profile image URL")
```

Comment:
We shouldn't need that if the data persisted to the database is the complete / built image url. It's just a matter of adding the model field to the `Meta.fields` attribute.

Reply:
@cacarrara Same thing, the DB field will not contain the complete URL. See: https://github.com/techmatters/terraso-backend/pull/74#discussion_r782353799